### ISSUE TITLE
Display statistics to users

### ISSUE THREAD

**Issue by naglepuff** [DigitalSlideArchive/wsi-superpixel-guided-labeling#3]
Helpful statistics to see while labeling superpixels may include

- Average prediction certainty
- Number of labeled superpixels of each category
- Average certainty per category
- Trending certainty over all iterations of training

These statistics should be displayed to the user while training. It might be worth storing these statistics to prevent re-computing them on hundreds of thousands of features.

**Comment by naglepuff** [comment]
A lack of these statistics could discourage users who are forced to correct a model's worst predictions. Showing users a positive trend when the activity is first loaded could alleviate this problem. The statistics could be shown the the user, but once they start labeling superpixels, they are hidden until the user wants to show them again.

**Comment by jagstein** [comment]
Number of laeled superpixels in each category
Number of labeled sueprpixels for each slide
Accuracy / uncertainty per category
Accuracy / uncertainty per slide

Something like this: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html

Plan now is to have some kind of hack (reports go to terminal or ???) but ultimately will need its own screen

**Comment by manthey** [comment]
We show a lot of this in review mode.
